Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -2243,6 +2242,26 @@ extension BrowserViewController {
             present(intro, animated: true)
             User.shared.hideRebrandIntro()
         }
```

Comment:
So rebrand intro pop up is still shown at the start for new users irrespective of when the default browser pop up is timed?

Reply:
Great question! In line 2255 in the same file, you'll see that there is an additional check inside `presentDefaultBrowserPromoIfNeeded()` for the Rebrand intro. So in that case it won't be shown.